### ISSUE TITLE
Getting error "Windows cannot find x, make sure you typed the name correctly, and then try again"

### ISSUE THREAD

**Issue by darthvader45** [MultiMC/MultiMC5#4096]
### Operating System

Windows

### Description of bug

For some reason, whenever I load an instance, go to the resource packs button and click it, then click 'open resource packs folder', I get a Windows error that says it cannot find the folder.  The game has never had this problem before today.  No idea why though.

### Steps to reproduce

1. Create and load a modded Forge instance (for me it happened with the latest 1.12.2 Forge).
2. Click Options, then Resource Packs
3. Click Open Resource Packs folder, and suddenly error.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Zetabite** [comment]
Please provide MultiMC-0.log, as file or as pastee link (or similar) link

**Comment by darthvader45** [comment]
You mean latest.log?  Alright.
[latest.log](https://github.com/MultiMC/MultiMC5/files/7150638/latest.log)

**Comment by Zetabite** [comment]
no. https://cdn.discordapp.com/attachments/531598137790562305/575381000398569493/unknown.png

**Comment by darthvader45** [comment]
https://paste.ee/p/Gf9bu

**Comment by Zetabite** [comment]
This doesn't look like an issue with MultiMC. If you want help with your modpack or mods, you can try asking in the modpack or mod's support channels/forums, or in the #minecraft channel in the [MultiMC Discord](https://discord.gg/multimc).

**Comment by darthvader45** [comment]
Huh, it was weirdly affecting all modded instances.

**Comment by Zetabite** [comment]
This is easier solved on the multimc discord

**Comment by phit** [comment]
there is no solution the game is just broken and it has nothing to do with multimc

**Comment by jamierocks** [comment]
>  I don't want to join the multimc discord for support, just put the answer here for everyone else having the issue please.

This is an issue tracker for MultiMC bugs, not game support. Please don't misuse it, the noise makes it less useful for it's actual purpose.

**Comment by SylveonBottle** [comment]
I've installed a modpack that I was having this issue with using the vanilla minecraft launcher.
The issue is not present. 

I tried the exact same modpack with MultiMC. The issue is present.
I've changed nothing between the two, only the launcher used.

If you are _absolutely sure_ that this issue has nothing to do with MultiMC, then I can only assume that it's up to the individual modpack creator to add support for MultiMC.
